src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -179,9 +165,28 @@
                      :title "Open Settings"
                      :active   (= @route-name :settings)}
              [:> Settings]]
+
             [button {:on-click #(dispatch [:modal/toggle])
                      :title "Choose Database"}
-             [:> LibraryBooks]]
+             [:div {:style {:display "flex"}}
+              [:> Storage {:style {:align-self "center"}}]
+              [:div {:style {:margin-left "-10px"
+                             :align-self "flex-end"}}
+               (cond
+                 (= @socket-status :closed)
+                 [:> Error (merge (use-style sync-icon-style)
```

Comment:
A possibly cleaner way to change the colors according to status would be to add a class on the button, `.closed`, `.running` etc., and change the color in the style object.